src/components/Exchange/PositionSeller.js

```diff
@@ -357,6 +360,14 @@ export default function PositionSeller(props) {
 
     receiveToken = isSwapAllowed && swapToToken ? swapToToken : collateralToken;
 
+    // receiveToken does not need to change for STOP order
+    if (isSwapAllowed && isUserAContractAddress && isAddressZero(receiveToken.address)) {
```

Comment:
Sorry, what about the case when user couldn’t change the token (orderOption === STOP)?

Reply:
As I understand it, in this case, if there is a long with collateral in ETH, then when triggerOrder is triggered by the actual user will get WETH or how will this happen?

Is there a similar check on the backend and should we give an explanation here?
@gdev8317

Reply:
Order book contract is not affected by this. So we don't need check for STOP order. I asked X earlier about this.